trajoptlib/include/trajopt/DifferentialTrajectoryGenerator.hpp

```diff
@@ -125,22 +132,24 @@ class TRAJOPT_DLLEXPORT DifferentialTrajectorySample {
    * @param heading The heading.
    * @param velocityL The left wheel velocity.
    * @param velocityR The right wheel velocity.
+   * @param omega The chassis angular velocity.
    * @param accelerationL The left wheel acceleration.
    * @param accelerationR The right wheel acceleration.
    * @param forceL The left wheel force.
    * @param forceR The right wheel force.
    */
   DifferentialTrajectorySample(double timestamp, double x, double y,
                                double heading, double velocityL,
-                               double velocityR, double accelerationL,
-                               double accelerationR, double forceL,
-                               double forceR)
+                               double velocityR, double omega,
```

Comment:
Swerve calls this angularVelocity.
```suggestion
                               double velocityR, double angularVelocity,
```